docs/fastdds/security/access_control_plugin/access_control_plugin.rst

```diff
@@ -0,0 +1,523 @@
+| **Action**                            | **XML element tag**                                                          |
++=======================================+==============================================================================+
+| Allow/Deny Publish                    | ``<publish>``                                                                |
++---------------------------------------+------------------------------------------------------------------------------+
+| Allow/Deny Subscribe                  | ``<subscribe>``                                                              |
++---------------------------------------+------------------------------------------------------------------------------+
+| Allow/Deny Relay                      | ``<relay>``                                                                  |
++---------------------------------------+------------------------------------------------------------------------------+
+
+Each action contains two conditions.
+
+* Allowed/Denied `Topics Condition`_
+* Allowed/Denied `Partitions Condition`_
+
+Topics Condition
+################
+
+This section is delimited by the ``<topics>`` XML element.
+It defines the Topic names that must be matched for the allow/deny rule to apply.
+Topic names may be given explicitly or by means of Topic name expressions.
+Each explicit topic name or Topic name expressions appears separately in a ``<topic>`` sub-element within the
+``<topics>`` element.
+
+The Topic name expression syntax and matching shall use the syntax and rules of the POSIX ``fnmatch()`` function as
+specified in
+
+.. literalinclude:: /../code/PermissionsTester.xml
+   :language: xml
+   :start-after: <!-->PERMISSIONS_TOPIC_CONDITION
+   :end-before: <!--><-->
+
+Partitions Condition
+####################
+
+This section is delimited by the ``<partitions>`` XML element.
+It limits the set Partitions names that may be associated with the (publish, subscribe, relay) action for the rule to
+apply.
+Partition names expression syntax and matching shall use the syntax and rules of the POSIX ``fnmatch()`` function as
+specified in `IEEE 1003.1-2017 <https://pubs.opengroup.org/onlinepubs/9699919799/functions/fnmatch.html>`_.
+If there is no ``<partitions>`` section within a rule, then the default "empty string" partition is assumed.
+
+.. literalinclude:: /../code/PermissionsTester.xml
+   :language: xml
+   :start-after: <!-->PERMISSIONS_PARTITION_CONDITION
+   :end-before: <!--><-->
+
+Signing documents using x509 certificate
+""""""""""""""""""""""""""""""""""""""""
+
+:ref:`domain_governance_doc` and :ref:`domainparticipant_permissions_doc` have to be signed using an X.509 certificate.
+Generation of an X.509 certificate is explained in :ref:`generate_x509`.
+Next commands sign the necessary documents for its use by the DDS\:Access\:Permissions plugin.
+
+.. code-block:: bash
```

Comment:
I general, in Lunix

Reply:
All openssl commands are the same for windows and linux.